packages/expo-modules-core/ExpoModulesCore.podspec

```
@@ -32,18 +32,18 @@ Pod::Spec.new do |s|
   s.dependency 'ReactCommon/turbomodule/core'
 
   if !$ExpoUseSources&.include?(package['name']) && ENV['EXPO_USE_SOURCE'].to_i == 0 && File.exist?("#{s.name}.xcframework") && Gem::Version.new(Pod::VERSION) >= Gem::Version.new('1.10.0')
```

I'm not sure where the `xcframework` file would be saved, but probably still in `ios` folder. It doesn't matter that much as I think we won't be able to prebuild `expo-modules-core` 😞

```suggestion
  if !$ExpoUseSources&.include?(package['name']) && ENV['EXPO_USE_SOURCE'].to_i == 0 && File.exist?("ios/#{s.name}.xcframework") && Gem::Version.new(Pod::VERSION) >= Gem::Version.new('1.10.0')
```